src/main/java/tutorial/Game.java

```diff
@@ -46,13 +53,18 @@ private boolean isComplete() {
         return towerOne.getDiscs().isEmpty() && towerTwo.getDiscs().isEmpty();
     }
 
+    //parameter types?
+    //negated method meaning?
     private boolean moveIsIllegal(int fromTower, int toTower) {
```

Comment:
Could this have Tower parameters and be expressed in the positive, i.e. `moveIsLegal`